todo_app/data/trello_items.py

```diff
@@ -0,0 +1,71 @@
+import requests
+import os
+import dotenv
+
+
+dotenv.load_dotenv()
+
+def get_items():
+    key =  os.getenv("TRELLO_API_KEY")
+    token =  os.getenv("TRELLO_API_TOKEN")
+    conn ="trello-proxy.azure-api.net"
+
+
+    headersList = {
+    "Accept": "*/*",
+    "User-Agent": "Thunder Client (https://www.thunderclient.com)" 
+    }
+
+    response =requests.get ("https://trello-proxy.azure-api.net/1/boards/6575b1e0697c0c91113c2035/lists?key={}&token={}&cards=open&card_fields=id,name".format(key,token),headers= headersList)
```

Comment:
This probably works for you, but it won't for me because I can't access your Trello board - we should pull out the id here and grab it from an environment variable:
```suggestion
    board_id = os.getenv("TRELLO_BOARD_ID")
    response =requests.get ("https://trello-proxy.azure-api.net/1/boards/{}/lists?key={}&token={}&cards=open&card_fields=id,name".format(board_id,key,token),headers= headersList)
```

Reply:
(This is still worth doing, although it's not a problem for you)